src/sinks/gcp_stackdriver_logging.rs

```diff
@@ -0,0 +1,320 @@
+use crate::{
+    event::{Event, Unflatten},
+    sinks::util::{
+        gcp::{GcpAuthConfig, GcpCredentials, Scope},
+        http::{https_client, HttpRetryLogic, HttpService},
+        tls::{TlsOptions, TlsSettings},
+        BatchBytesConfig, Buffer, SinkExt, TowerRequestConfig,
+    },
+    topology::config::{DataType, SinkConfig, SinkContext, SinkDescription},
+};
+use futures::{stream::iter_ok, Future, Sink};
+use http::{Method, Uri};
+use hyper::{Body, Request};
+use lazy_static::lazy_static;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Deserialize, Serialize, Debug, Clone, Default)]
+#[serde(deny_unknown_fields)]
+pub struct StackdriverConfig {
+    #[serde(flatten)]
+    pub log_name: StackdriverLogName,
+    pub log_id: String,
+
+    pub resource: StackdriverResource,
+
+    #[serde(flatten)]
+    pub auth: GcpAuthConfig,
+
+    #[serde(default, flatten)]
+    pub batch: BatchBytesConfig,
+    #[serde(flatten)]
```

Comment:
This should be just default.
```suggestion
    #[serde(default)]
```

Reply:
Oops, right, will fix.